### ISSUE TITLE
Publish doesn't work on build 0.16.19969

### ISSUE THREAD

**Issue by ajkauffmann** [Microsoft/AL#2867]
When using AL:Publish with build 0.16.19969 I get an error message
 
![image](https://user-images.githubusercontent.com/19155831/42576523-deb2828c-8522-11e8-9344-1d15d338f50c.png)

Messages in the debug console:
```
[2018-07-11 15:55:30.68] Sending request to http://bcsandbox:7049/nav/dev/apps?SchemaUpdateMode=synchronize
[2018-07-11 15:55:31.75] Success: The package 'Default publisher_ALProject49_1.0.0.0.app' has been published to the server.
[2018-07-11 15:55:31.83] Error: An internal error has occurred

```

At the same time, Publish without debugging works fine. So it seems to be related to the debugger.
Reproduced both on local sandbox and on Azure VM. 
Build 0.16.19969 (microsoft/sandbox)

**Comment by thpeder** [comment]
Hi @ajkauffmann,
Do you have anything in the event viewer?

**Comment by ajkauffmann** [comment]
Checked that as well, nothing found.

I can share a VM if that helps.

**Comment by thpeder** [comment]
Hi @ajkauffmann,
You could send them to thpeder and I'll have a look.

**Comment by Fkoehnen** [comment]
Can confirm with the june developer preview on Azure.
Have just created the AL: Go! example and gotten the same error message as @ajkauffmann when trying to debug.

**Comment by AndreaAntolini** [comment]
Hi all,
I have the same exact issue with a newly created D365BC trial...

**Comment by thpeder** [comment]
There is a problem with the docker image but we are trying to locate the problem and provide a new image once we know the cause.

**Comment by bvbeek** [comment]
Same issue here, any news about this issue?

**Comment by Enomis82** [comment]
any news about this?
Thks

**Comment by dennisfrostlander** [comment]
This is the problem on the Nav server in the 12.2 builds. Debugger fails to launch when UsernamePassword authentication method is used.
The issue should be already resolved on bcinsider builds (12.3).
